geoportailv3/templates/index.html

```diff
@@ -196,7 +197,9 @@ <h2 translate>infos</h2>
     <script src="${request.route_url('deps.js')}"></script>
     <script src="${request.static_url('geoportailv3:static/js/main.js')}"></script>
     <script src="${request.static_url('%s/ngeo/utils/watchwatchers.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/proj4/dist/proj4-src.js' % node_modules_path)}"></script>
 % else:
+    <script src="${request.static_url('%s/proj4/dist/proj4-src.js' % node_modules_path)}"></script>
```

Comment:
As I said already, we want to add proj4 to `build.js`. But we can do that with a separate PR if you do not feel comfortable with it.